README.md

```diff
@@ -1,40 +1,86 @@
+
+```
+text: PropTypes.string.tag({
+      name: 'Main Alert Text'
+    }).isRequired
+```
+
+5. Create two custom fields to allow the user to add an external link. One field should be for the link text; the other for the url. This second one should be `PropTypes.url`, which validates the url in PageBuilder. Put these two in one **group** in the admin. 
+
+```
+linkText: PropTypes.string.tag({
+      group: 'Call to Action',
+      name: 'Text',
+      description: 'Text to prompt the users to click the link',
+    }),
+    link: PropTypes.url.tag({
+      group: 'Call to Action',
+      name: 'Url',
+    })
+```
+
+## Inline editing
+
+
 On this branch, we have updated our `/sample/color-block` feature to offer PageBuilder users the ability to customize the feature using Custom Fields.
 
-So far, we have updated our output type to modify our head and footer on the page, created a basic layout to provide semantic html to our page, and began creating and modifying features using custom fields from Fusion's component context. Let's practice with an exercise - creating an alert bar! Switch to `exercise-01` to see one example implementation. Feel free to continue here once you have completed the exercise.
+So far, we have updated our output type to modify our head and footer on the page, created a basic layout to provide semantic html to our page, and began creating and modifying features using custom fields from Fusion's component context. 
 
-In this lab, we will allow the user to directly customize my feature using inline editing for a streamlined experience. Building on the alert bar feature from `exercise-01` we provide a clearer and easier way for editors to configure custom fields through inline editing as documented [here](https://redirector.arcpublishing.com/alc/arc-products/pagebuilder/fusion/documentation/recipes/adding-custom-fields.md#inline-editing)
+Now we will allow the user to directly customize my feature using inline editing for a streamlined experience. Building on the alert bar feature we just created, we provide a clearer and easier way for editors to configure custom fields through inline editing as documented [here](https://redirector.arcpublishing.com/alc/arc-products/pagebuilder/fusion/documentation/recipes/adding-custom-fields.md#inline-editing)
 
-1. Complete the user story from branch `exercise-01`. This includes adding a new section to the layout and a new feature in the `global/alert-bar` directory. After completing the exercise, return here and we will modify it by adding inline editing to a text field in the feature. You can also complete the steps below on the custom field we added to the color block.
+6. Make sure the above ticket is completed. This includes adding a new section to the layout and a new feature in the `global/alert-bar` directory. Now we will modify it by adding inline editing to a text field in the feature. You can also complete the steps below on the custom field we added to the color block.
```

Comment:
`Make sure the above ticket is completed` can be removed since its all 1 lab now

Reply:
:+1: missed that phrasing